contracts/voting-verifier/src/state.rs

```diff
@@ -69,13 +77,28 @@ impl Poll {
     }
 }
 
+#[cw_serde]
+pub struct PollMessage {
+    pub msg: Message,
+    pub poll_id: PollID,
+    pub index_in_poll: usize,
+}
```

Comment:
why is this now necessary?

Reply:
Before, we had in storage the `PENDING_MESSAGES` which were moved to the `VERIFIED_MESSAGES` when the poll was finished and if they were verified, but now messages verification is calculated on the fly when the query is called so we don’t need those storage constants anymore, instead since the verification is calculated on the fly regardless of if the poll is finished or not (for early verification) wee need a way to locate a given message inside a poll to check for it’s verification status. The query receives the message itself and not a poll, so I created POLL_MESSAGES which use this struct to link a message to its location in a poll. This is where it’s being used: https://github.com/axelarnetwork/axelar-amplifier/blob/692c64508eee9a4bb7de518cd37a2fdc4b43f3d6/contracts/voting-verifier/src/query.rs#L44